honeybee_radiance/cli/multiphase.py

```diff
@@ -112,3 +112,87 @@ def view_matrix_command(
         sys.exit(1)
     else:
         sys.exit(0)
+
+
+@multi_phase.command("daylight-matrix")
```

Comment:
Should we call this daylight coefficient instead? I think that's technically what we are calculating here - or even view coefficient. It's not really daylight coefficient depending on how it is being used.

What do you think @built-tools?

Reply:
@mostaphaRoudsari, you are right that the name can be more generic as the command is essentially the most simple use of rfluxmtx. As such I changed the name to 'flux-transfer'.